ci/plugins/cloudtest/hooks/post-command

```diff
@@ -75,6 +75,10 @@ kubectl get events > kubectl-get-events.log || true
 kubectl get all > kubectl-get-all.log || true
 kubectl describe all > kubectl-describe-all.log || true
 
+kubectl -n kube-system get events > kubectl-get-events-kube-system.log || true
+kubectl -n kube-system get all > kubectl-get-all-kube-system.log || true
+kubectl -n kube-system describe all > kubectl-describe-all-kube-system.log || true
```

Comment:
Wondering if we should consolidate these statements with the three previous ones and use `--all-namespaces`. In the cloud repo, several different namespaces are used.